redisraft.h

```diff
@@ -579,6 +585,8 @@ typedef struct {
 #define CMD_SPEC_WRITE          (1<<2)      /* Command is a (potentially) write command */
 #define CMD_SPEC_UNSUPPORTED    (1<<3)      /* Command is not supported, should be rejected */
 #define CMD_SPEC_DONT_INTERCEPT (1<<4)      /* Command should not be intercepted to RAFT */
+#define CMD_SPEC_SORTABLE       (1<<5)      /* Command output should be sorted within a lua script */
```

Comment:
Suggest a more accurate name: `CMD_SPEC_SORT_REPLY`.

Reply:
ok.  I thought sortable was fine, as its a command whose output we can sort without breaking its semantics.  but I changed it to yours